content/200-concepts/100-components/02-prisma-client/260-client-extensions/140-shared-extensions.mdx

```diff
@@ -7,50 +7,62 @@ tocDepth: 4
 
 <TopBlock>
 
-You can share your [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) with other users, either as packages or as code snippets, and import extensions that other users create into your Prisma projects.
+You can share your [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) with other users, either as packages or as modules, and import extensions that other users create into your project.
+
+If you would like to build a shareable extension, we also recommend using the [`prisma-client-extension-starter`](https://github.com/prisma/TODO) template.
 
 </TopBlock>
 
 ## Share Prisma Client extensions
 
-When you want to create extensions that will work for other users, and not just for your specific schema, use the `$all` prefix, such as [`$allModels`](/concepts/components/prisma-client/client-extensions/model#add-a-custom-method-to-all-models-in-your-schema). Extensions that use `$all` apply generally, not just to a named component of a specific schema.
+When you want to create extensions other users can use, and not tailored just for your schema, Prisma provides utilities to allow you to create shareable extensions.
+
+### Define an extension
+
+Use the `Prisma.defineExtension` method to make your extension shareable. You can use it to package the extension to either separate your extensions in multiple files or share it with other users as an npm package.
```

Comment:
"multiple files" (as in the extension is in more than 1 file) or "a separate file" (as in not the file where your other code lives)?